src/SFA.DAS.Payments.Monitoring.Metrics.Data/DcMetricsDataContext.cs

```diff
@@ -268,14 +390,30 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
 
         public async Task<List<TransactionTypeAmounts>> GetEarnings(long ukprn, short academicYear, byte collectionPeriod, CancellationToken cancellationToken)
         {
+            if (academicYear == 2526)
```

Comment:
introduces technical debt and requires a follow up ticket to fix after R03.  Would be better to have code that used the new sql from 2526 onwards.